async/kafka/kafka.go

```diff
@@ -23,12 +23,32 @@ const (
 )
 
 var topicPartitionOffsetDiff *prometheus.GaugeVec
+var countAcks prometheus.Counter
```

Comment:
countAcks and nack can be one counter with a label differentiator.
same goes for received, decoded, decoder error...
no need to have separate counter for each of whose. we should group them together